### PR TITLE
added op counter 

### PR DESCRIPTION
**Pull request by MatthiasLeeIBM**
* added optional op counter which bails workers after a set number of operations has been completed

Bugz: 85691

### CODE REVIEW

**Comment by ksnavely** [comment]
Would you add a brief word to the README.md on configuration and functionality of this feature?

**Comment by MatthiasLeeIBM** [comment]
will do

File: src/basho_bench_worker.erl [hunk 1]
```diff
@@ -257,11 +258,29 @@ worker_idle_loop(State) ->
 
 %% Traditional call to run/4 passing OpTag, keygen, valgen, and driver state
 worker_next_op2(#state{api_pass_state=false}=State, OpTag) ->
-    catch (State#state.driver):run(OpTag, State#state.keygen, State#state.valgen,State#state.driver_state);
+    %% call operation counter
```
Comment (ksnavely) [inline src/basho_bench_worker.erl]: I'm not sure the `call operation counter` comments are adding much.

**Comment by ksnavely** [comment]
+1

**Pull request merged**